src/Commands/ToggleLifter.cpp

```diff
@@ -1,12 +1,13 @@
 #include "ToggleLifter.h"
+#include "../Robot.h"
 
```

Comment:
Consider adding `#include ../Subsystems/Shooter.h` since we are using its enum LifterValue